From 5d63b1b64d1d52dc78423ef05c869323d0580045 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:59:07 -0400
Subject: [fs] btrfs: remove field tree_mod_seq_elem from btrfs_fs_info struct

Message-id: <1407179109-13714-61-git-send-email-sandeen@redhat.com>
Patchwork-id: 87296
O-Subject: [RHEL7.1 PATCH 060/422] Btrfs: remove field tree_mod_seq_elem from btrfs_fs_info struct
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Filipe David Borba Manana <fdmanana@gmail.com>

commit e223cfcd3eed7401e4217c7fb7391017c8124d8c
Author: Filipe David Borba Manana <fdmanana@gmail.com>
Date:   Fri Dec 13 19:39:54 2013 +0000

    Btrfs: remove field tree_mod_seq_elem from btrfs_fs_info struct

    It's not used anywhere, so just drop it.

    Signed-off-by: Filipe David Borba Manana <fdmanana@gmail.com>
    Signed-off-by: Josef Bacik <jbacik@fb.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/ctree.h b/fs/btrfs/ctree.h
index a924274..3c9053a 100644
--- a/fs/btrfs/ctree.h
+++ b/fs/btrfs/ctree.h
@@ -1462,7 +1462,6 @@ struct btrfs_fs_info {
  spinlock_t tree_mod_seq_lock;
  atomic64_t tree_mod_seq;
  struct list_head tree_mod_seq_list;
- struct seq_list tree_mod_seq_elem;
 
  /* this protects tree_mod_log */
  rwlock_t tree_mod_log_lock;
-- 
1.7.1